style.css

```diff
@@ -583,10 +583,6 @@ q:after {
 # Forms
 --------------------------------------------------------------*/
 
-/*--------------------------------------------------------------
-# Forms
---------------------------------------------------------------*/
-
```

Comment:
This was fixed in https://github.com/WordPress/twentyseventeen/commit/033c71099840c0462c7f48c2addcbbfffc00ebf2